src/routes/GroupSettings/RelatedGroupsTab/RelatedGroupsTab.js

```diff
@@ -290,13 +289,13 @@ export function GroupCard ({ actionMenu, thisGroup, group, questionAnswers, type
       thisGroup.groupToGroupJoinQuestions &&
       thisGroup.groupToGroupJoinQuestions.map(q =>
         <div styleName='answer' key={q.id}>
-          <h3>{q.text}</h3>
+          <div styleName='subtitle'>{q.text}</div>
           <p>{get('answer', questionAnswers && questionAnswers.find(qa => qa.question.id === q.questionId)) || <i>Not answered</i>}</p>
         </div>
       )}
       {otherAnswers.map(qa =>
         <div styleName='answer' key={qa.id}>
-          <h3>{qa.question.text}</h3>
+          <div styleName='subtitle'>{qa.question.text}</div>
```

Comment:
why the preference for the styleName over h3?

Reply:
Our reset / stylesheet is not consistently applied for the standard HTML tags. I prefer them, but as they are globally applied and we are using modules in moving towards normalizing this (and User) settings areas I think for now we should be moving away from them and towards a shared module stylesheet across all these screens. Right now it's somewhere 1/2 between. It was more chaotic before I got in there this round :) Moving in increments. Hope that makes sense. In short, it's a little bit of a mess and moving away from the `hX` headers for now is likely to make things a little more clear overall than less. IMHO